services/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -113,7 +113,7 @@
   private final Consumer<ServiceDiscoveryConfig.Builder> discoveryOptions;
 
   private Microservices(Builder builder) {
-    this.id = IdGenerator.generateId();
+    this.id = UUID.randomUUID().toString();
```

Comment:
What is the motivation for these changes?

Reply:
we removed utils-module from the current version of the cluster and right now I don't want to add a dependency only for generation id.